.eslintrc

```diff
@@ -18,8 +18,13 @@
       "files": [ "**/*.ts" ],
       "extends": [
         "eslint:recommended",
-        "prettier"
-      ]
+        "prettier",
+        "plugin:@typescript-eslint/eslint-recommended",
```

Comment:
Can we maybe create a separate issue for es-lint and resolve it within it? So we can discuss it more focussed?

Reply:
Answered in main converstion thread:
_"The eslint changes belong to this PR because the `var` issues in the three files were fixed automatically by eslint with this configuration..."_